### DISCUSSION TITLE
想问下2.x的计划和里程碑。

### DISCUSSION THREAD

**Discussion by gxy01** [alibaba/formily#1144] [Ideas]
我们有个新项目打算用formily。那么现在我应该用1.x还是等不久就能直接用2呢？

**Comment by janryWang** [comment]
2.x正式版在2021年年底发布，beta版本会在2月底发布

**Reply by wittech** [reply]
beta版能用到生产？会不会后续还有大的变动呢？

**Comment by UlricGan** [comment]
请问最新的beta版什么时候发布到npm，我看2.0.0-beta.2 是2020年6月份的
